test/integration/helpers/mswjs/browserWorker.js

```diff
@@ -25,6 +25,10 @@ const createWorker = () => {
     return originalUse.apply(worker, handlers);
   };
 
+  worker.forceUse = (...handlers) => {
```

Comment:
I added this method for the case when we want to mock the responses all the time. For this test, I do not have any code in production that could generate an event history operation.